caffe2/opt/onnxifi_transformer.cc

```diff
@@ -84,7 +97,8 @@ void FillModelInfo(::ONNX_NAMESPACE::ModelProto* model) {
 }
 } // namespace
 
-OnnxifiTransformer::OnnxifiTransformer(bool debug) : debug_(debug) {
+OnnxifiTransformer::OnnxifiTransformer(bool infer_shapes, bool debug)
```

Comment:
Probably consider creating a "Config" struct to store these configuration parameters, othrewise everytime we add new one will need to touch the whole chain of function calls.

Reply:
Yes. My tolerance is 3-4. :)